EXT_BeamPhysics.md

```diff
@@ -0,0 +1,344 @@
+BeamPhysics Extension to the openPMD Standard for Describing Particle Beams, Photons, and External Fields
+=========================================================================================================
+
```

Comment:
As in `ED-PIC` and `SpeciesType`: can we please make
```md
openPMD extension name: `BeamPhysics`
```

The first line here? Just to keep extensions consistent.
```suggestion

openPMD extension name: `BeamPhysics`
```

Reply:
Will be in next version